### PR TITLE
Or 1139 deploy fee token contracts

### PR DESCRIPTION
**Pull request by theo-learner**
# Changes
add TON, TON_FeeVault contracts
* TON (only hardhat): L1 TON contract
* TON_FeeVault: manage price ratio and withdraw TON to l1 fee wallet 
  * hardhat: pre-deployed contract
  * live: deploy contract in live l2

add L2StandardERC20, Ton_FeeVault in take-dump (only dev)
add deploy script for L1 TON 
add deploy script for TON_FeeVault (only live network)

# How to test
```bash
# you should build the project first.

# modify .env.example (local)
CONTRACTS_TARGET_NETWORK=local
CONTRACTS_DEPLOYER_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
CONTRACTS_RPC_URL=http://localhost:9545

# apply to .env
cp .env.example .env

# deploy contracts (TON_FeeVault is dumped in genesis)
npx hardhat deploy --network local

# deploy TON_FeeVault (live network)
# To verify, you use ETHERSCAN_API_KEY, AUTOMATICALLY_TRANSFER_OWNERSHIP in .env
npx hardhat deploy --network titan-goerli-nightly --tags l2
```

### CODE REVIEW

File: packages/tokamak/contracts/deploy/019-Tokamak-TONFeeVault.deploy.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* Imports: External */
```
Comment (rlgns98kr) [inline packages/tokamak/contracts/deploy/019-Tokamak-TONFeeVault.deploy.ts:1]: I think if we execute `hardhat deploy --network mainnet`, this file will execute. Do you know how can we avoid it? Or, can you share other details?

File: packages/tokamak/contracts/contracts/L1/token/TON.sol [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
```
Comment (rlgns98kr) [inline packages/tokamak/contracts/contracts/L1/token/TON.sol:1]: I think we can assume there is TON contract in L1. If you use this only for tesing and will be removed in production. It's okay for me! 🙇🏼‍♂️

File: packages/tokamak/contracts/deploy/019-Tokamak-TONFeeVault.deploy.ts (hunk 1 above)
Comment (rlgns98kr) [reply packages/tokamak/contracts/deploy/019-Tokamak-TONFeeVault.deploy.ts:1]: Great! only for liveNetworks. Thank you very much!

**Review by steven94kr — APPROVED** [review]
Thank you!

**Pull request merged**